Uzduotis16/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+
+namespace Uzduotis16
+{
+    internal class Program
+    {
+        static void Main()
+        {
+            // Raskite visų skaičių nuo 1 iki 100 sumą.
+           
+            int suma = 0;
+
+            for (int i = 1; i < 101; i++)
+            {
+                suma += i;
+            }
+
+            Console.WriteLine(suma);
+
+            //Raskite visų lyginių skaičių nuo 20 iki 40 sumą.
+
+            for (int i = 20; i < 41; i++)
```

Comment:
Kadangi naudojate tą patį kintamąjį `suma`, tai prieš kiekvieną `for` ciklą šį kintamąjį reikia "atstatyti" (priskirti jam reikšmę 0). Nes dabar kaupiate sumą toliau ir gaunate neteisingus atsakymus.